std/file.d

```diff
@@ -4618,13 +4622,13 @@ enum SpanMode
 
     alias removeRoot = (e) => e.relativePath(root);
 
-    root.dirEntries(SpanMode.shallow).map!removeRoot.equal(
+    root.dirEntries(SpanMode.shallow).array.sort.map!removeRoot.equal(
         ["plants", "animals"]);
 
-    root.dirEntries(SpanMode.depth).map!removeRoot.equal(
+    root.dirEntries(SpanMode.depth).array.sort.map!removeRoot.equal(
         ["plants", "animals/dog", "animals/cat", "animals"]);
 
-    root.dirEntries(SpanMode.breadth).map!removeRoot.equal(
+    root.dirEntries(SpanMode.breadth).array.sort.map!removeRoot.equal(
         ["plants", "animals", "animals/dog", "animals/cat"]);
```

Comment:
This gets problematic here as this is an example. Suggestions to make this unittest example clear?

Reply:
Why not just wrap the line in `assert(...)`?

When rendered on the website, it will be converted to a `writeln`, with the expected result in a comment.

Reply:
Because `root.dirEntries(SpanMode.depth)` and `root.dirEntries(SpanMode.breadth)` sorted are the same but the example what to show the difference between them.